app/dashboard/models.py

```diff
@@ -2198,21 +2198,14 @@ def to_dict(self, activities=True, leaderboards=True, network=None, tips=True):
         }
 
         if activities:
-            fulfilled = self.fulfilled.filter(
+            all_activities = self.activities.filter(
```

Comment:
Did you decide to exclude tips for now?

Reply:
The tips we are already sending in a non activity way. So i am excluding it in the activities.